java/src/jmri/jmrix/jinput/TreeModel.java

```diff
@@ -48,7 +48,9 @@ private TreeModel() {
         // needed to get the display to start
         // insertNodeInto(new UsbNode("System", null, null), dRoot, 0);
         // start the USB gathering
-        (new Runner()).start();
+        Runner r = new Runner();
+        r.setName("TreeModel loader");
```

Comment:
Just out of curiosity, would it be preferable to set these names in the constructor for named classes like this or after constructing the class?